Composer/packages/client/__tests__/components/CreationFlow/CreateOptions/index.test.tsx

```diff
@@ -33,7 +34,9 @@ describe('<CreateOptions/>', () => {
     return renderWithRecoil(
       <CreateOptions
         fetchReadMe={handleFetchReadMeMock}
+        localTemplatePath={''}
         path="create"
+        setLocalTemplatePath={setLocalTemplatePathMock}
```

Comment:
this is a callback, name it accordingly: onUpdateLocalTemplatePath

Reply:
There are other similar cases I didn't comment on but please fix.

Reply:
Handled